contract/area_types.go

```diff
@@ -2,10 +2,11 @@ package contract
 
 // AreaType is an area-type model with ID and Label
 type AreaType struct {
-	ID          string `json:"id"`
-	Label       string `json:"label"`
-	Description string `json:"description"`
-	TotalCount  int    `json:"total_count"`
+	ID                      string `json:"id"`
+	Label                   string `json:"label"`
+	Description             string `json:"description"`
+	TotalCount              int    `json:"total_count"`
+	GeographyHierarchyOrder int    `json:"geography_hierarchy_order"`
```

Comment:
This could be called `HierarchyOrder` as Geography is implied by it being part of AreaType

Reply:
This has now been changed.